profiles.py

```diff
@@ -0,0 +1,30 @@
+from spectrum import load, input, checks
+from sys import argv, exit
+import fileinput
+import requests
+
+JOURNAL = checks.JOURNAL.with_resource_checking_method('head').with_query_string("open-sesame")
+
+
+if __name__ == '__main__':
+    api_checks = bool(argv[2]) if len(argv) > 2 else False
+    limit = load.Limit(argv[3] if len(argv) > 3 else None)
+    load.LOGGER.info("Setting iterations limit %s", limit)
+
+    pages = []
```

Comment:
better to divide this into two cases:
- check API (smoke tests mostly)
- check Journal (truly end-to-end)